src/lib.rs

```diff
@@ -0,0 +1,437 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+#![warn(rust_2018_idioms, unused_lifetimes, unused_qualifications, clippy::all)]
+
+#[macro_use]
+extern crate anyhow;
```

Comment:
pretty sure this is redundant in Rust 2021, just import directly whatever you need to import like everywhere else

Reply:
Fixed.